action.yml

```diff
@@ -7,6 +7,11 @@ inputs:
   configuration-path:
     description: 'The path for the label configurations'
     default: '.github/labeler.yml'
+  not-found-label:
+    description: 'The label to apply when no matches were found'
+  operations-per-run:
+    description: 'The maximum number of operations per run, used to control rate limiting'
+    default: 30
```

Comment:
I don't think this is the correct way to implement rate limiting. Doesn't the Actions library handle rate limiting for you?

Reply:
Got from another official action. I think it’s actions/stale